src/ape/api/accounts.py

```diff
@@ -268,7 +263,7 @@ def __len__(self) -> int:
         """
 
     @abstractmethod
-    def __iter__(self) -> Iterator[AccountAPI]:
+    def __iter__(self):
```

Comment:
mypy gets very angry about leaving this as `Iterator[AccountAPI]`. I tried a few type alternatives but the pydantic BaseModel seems to take precedent returning a type of `Generator[Tuple[str, Any], None, None]`.

Reply:
Is it bad to be overriding this method? It sounds like the base model is already maybe doing something with it?

Reply:
> Is it bad to be overriding this method? It sounds like the base model is already maybe doing something with it?

It is actaully, for serialization. We should probably consider an alternative method rather than overriding this